src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrPatchHandler.cs

```diff
@@ -268,10 +319,32 @@ public async Task ApplyPatch(
 
         args.Add(patch.Path);
 
-        var result = await _processManager.ExecuteGit(targetDirectory, args, cancellationToken: CancellationToken.None);
-
+        var result = await repo.ExecuteGitCommand([..args], cancellationToken: CancellationToken.None);
         if (!result.Succeeded)
         {
+            // When we are applying and wish to keep conflicts in the working tree, we need to clean up a bit:
+            if (keepConflicts)
+            {
+                // Collect the list of conflicted files:
+                cancellationToken.ThrowIfCancellationRequested();
+                var conflictedFilesResult = await repo.ExecuteGitCommand(["diff", "--name-only", "--diff-filter=U"], CancellationToken.None);
```

Comment:
we could make this it's own method, returning the list of files, I think we also use the same code in the Meaningless Changes dedection

Reply:
This one is a bit different and only returns conflicted files (in the conflicted state)